### PR TITLE
Add scatter region RPC

### PR DESCRIPTION
**Pull request by huachaohuang**
Add scatter region RPC to scatter regions across the cluster.

### CODE REVIEW

File: src/pd/client.rs [hunk 1]
```diff
@@ -413,4 +413,24 @@ impl PdClient for RpcClient {
             .request(req, executor, LEADER_CHANGE_RETRY)
             .execute()
     }
+
+    fn scatter_region(&self, mut region: RegionInfo) -> Result<()> {
```
Comment (siddontang) [inline src/pd/client.rs]: only use in test?

Comment (huachaohuang) [reply src/pd/client.rs]: No, it will be used in the import service.

**Pull request merged**